pkg/iac/supplier/supplier.go

```diff
@@ -34,10 +35,11 @@ func GetIACSupplier(configs []config.SupplierConfig,
 	library *terraform.ProviderLibrary,
 	backendOpts *backend.Options,
 	progress output.Progress,
+	alerter *alerter.Alerter,
 	factory resource.ResourceFactory,
 	filter filter.Filter) (resource.Supplier, error) {
 
-	chainSupplier := resource.NewChainSupplier()
```

Comment:
Is NewChainSupplier still used somewhere ?

Reply:
right it might have disapeared with the rebase I was sure it had another usage...

Reply:
The old `ChainSupplier` appears to be unused anymore, should we delete it ?